File: DajiOrder_java/src/Entity/Application/Ack.java
package Entity.Application;

import Utility.Common;

/**
 * Created by Bianshuai on 2017/1/8.
 */
public class Ack {

    //ID
    public String Id = null;

    // 是否成功
    public Boolean IsSuccess = false;

    //状态编号
    public int StatusCode = 0;

    // 信息
    public String Message = null;

    public Ack() {
        Id = Common.CreateGuid();
        IsSuccess = true;
        Message = "";
    }
}

File: ScalaWeb/src/Entity/Application/Ack.scala
package Entity.Application

import Utility.Common

/**
  * Created by Bianshuai on 2017/1/18.
  */
class Ack {

  //ID
  var Id: String = Common.CreateGuid()

  // 是否成功
  var IsSuccess: Boolean = true

  //状态编号
  var StatusCode: Int = 0

  // 信息
  var Message: String = ""
}
